lib/go-atscfg/logsdotxml.go

```diff
@@ -27,12 +27,18 @@ import (
 const LogsXMLFileName = "logs_xml.config"
 const ContentTypeLogsDotXML = `text/xml`
 
+const LineCommentLogsDotXML = `<!--`
+
 func MakeLogsXMLDotConfig(
 	profileName string,
 	paramData map[string]string, // GetProfileParamData(tx, profile.ID, LoggingYAMLFileName)
 	toToolName string, // tm.toolname global parameter (TODO: cache itself?)
 	toURL string, // tm.url global parameter (TODO: cache itself?)
 ) string {
+
+	// Note LineCommentLogsDotXML must be a single-line comment!
+	// But this file doesn't have a single-line format, so we use <!-- for the header and promise it's on a single line
+	// Note! if this file is ever changed to have multi-line comments, LineCommentLogsDotXML will have to be changed to the empty string.
 	hdrComment := GenericHeaderComment(profileName, toToolName, toURL)
 	hdrComment = strings.Replace(hdrComment, `# `, ``, -1)
 	hdrComment = strings.Replace(hdrComment, "\n", ``, -1)
```

Comment:
This should probably just replace `# ` with `<!--` and `\n` with `-->\n`? That would make it at least work in the future if someone returns a multiline comment. This works, though, doesn't change current behaviour, and won't break in the future. This is fine, though.